Android-Sample/README.md

```diff
@@ -14,7 +13,8 @@ the session:
 * The client is unable to publish.
 
 The sample app only subscribes to the test stream. It does not subscribe to other streams in the
-session.
+test session. Use a separate OpenTok session (and session ID) to share audio-video streams between
+clients.
```

Comment:
I would also include the following sentence
"Do not use the test session ID for your actual call. Use a separate OpenTok session (and session ID) to share audio-video streams between clients."